utilities/transactions/write_prepared_txn_db.h

```diff
@@ -802,6 +809,11 @@ class WritePreparedCommitEntryPreReleaseCallback : public PreReleaseCallback {
         db_->AddCommitted(prep_seq_ + i, last_commit_seq);
       }
     }  // else there was no prepare phase
+    if (includes_aux_batch_) {
+      for (size_t i = 0; i < aux_batch_cnt_; i++) {
+        db_->AddCommitted(aux_seq_ + i, last_commit_seq);
```

Comment:
Is the calculation of `last_commit_seq` still correct in the presence of aux_seq_ (especially with data_batch_cnt_ > 1 case)? Can we add a comment explaining this?